tests/test_utils.py

```diff
@@ -58,3 +64,21 @@ def test_utils_load_pkg_resource_fallback(self):
         self.assertIsInstance(data[0]["os"], str)
         self.assertIsInstance(data[0]["os_version"], str)
         self.assertIsInstance(data[0]["platform"], str)
+
+        # cleanup
+        unload_module("fake_useragent")
+        sys.path.remove(zip_filename)
+
+        try:
+            d.cleanup()
+        except PermissionError:
```

Comment:
If `with TemporaryDirectory() as d:` can't be used due to Windows (I can't test), please wrap all of the code in a `try: finally:` so that cleanup is done regardless of other errores.